src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/config.jelly

```diff
@@ -1,18 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials" xmlns:scm="/jenkins/scm/api/form">
-  <j:choose>
-    <j:when test="${descriptor.apiUriSelectable}">
-      <f:entry title="${%API endpoint}" field="apiUri">
-        <f:select/>
-      </f:entry>
-    </j:when>
-    <j:otherwise>
-      <f:entry title="${%API endpoint}" field="apiUri">
-        <f:textbox readonly="readonly"/>
-      </f:entry>
-    </j:otherwise>
-  </j:choose>
+  <f:entry title="${%API endpoint}" field="apiUri">
+    <f:select/>
+  </f:entry>
```

Comment:
I'm pretty sure this drop down will also include `github.com` if the endpoint list is empty. Or did that change at some point?

Reply:
it does not include github.com